cmd/juju/commands/bootstrap.go

```diff
@@ -1497,6 +1498,21 @@ func (c *bootstrapCommand) bootstrapConfigs(
 		return bootstrapConfigs{}, errors.Annotate(err, "constructing bootstrap config")
 	}
 
+	// Pre-process controller attributes.
+	if _, ok := controllerConfigAttrs[controller.CAASOperatorImagePath]; ok {
+		return bootstrapConfigs{}, fmt.Errorf("%q is no longer supported controller configuration",
```

Comment:
Should we provide a reference to watch is in place now or some more documentation.

Reply:
I've got a jira ticket to update docs for caas-image-repo.